group.go

```diff
@@ -864,3 +864,54 @@ func (cli *Client) parseGroupNotification(node *waBinary.Node) (interface{}, err
 		return groupChange, nil
 	}
 }
+
+// GroupJoinApprovalMode sets the group join approval mode to 'on' or 'off'.
+func (cli *Client) GroupJoinApprovalMode(jid types.JID, mode string) error {
+	if mode != "on" && mode != "off" {
```

Comment:
This looks like it should be a boolean rather than a string

Reply:
it's done